README.md

```diff
@@ -1 +1,82 @@
-# angular-track-by-readme
\ No newline at end of file
+# Track by
+
+## Overview
+
+Track by allows Angular to speed up our `ng-repeat`s massively - let's take a look how.
+
+## Objectives
+
+- Describe "track by" clause
+- Write an ng-repeat that utilises "track by"
+
```

Comment:
Add a short introduction here about performance issues in general and that Angular has built in tools to address some of them. Mention that now that we have our application up and running, we can start to think about scaling it, etc.